plugins/repository-azure/qa/microsoft-azure-storage/build.gradle

```diff
@@ -71,6 +71,7 @@ testClusters.integTest {
         // in a hacky way to change the protocol and endpoint. We must fix that.
         setting 'azure.client.integration_test.endpoint_suffix',
                 { "ignored;DefaultEndpointsProtocol=http;BlobEndpoint=http://${azureStorageFixture.addressAndPort }" }
+        setting 'azure.threadpool.size', '1'
```

Comment:
why test just with `1`?

Reply:
I figured since the 3rd party tests run using the defaults it wouldn't be the worst idea to test the extremal case here?